### PR TITLE
fix(RHINENG-2741): disables zero state when there is edge devices

### PR DESCRIPTION
**Pull request by mkholjuraev**
# Description

Associated Jira ticket: # [(issue)](https://issues.redhat.com/browse/RHINENG-2741)

Shows the zero state only when there is no both conventional and immutable systems.

Also, improves the API requests scattered around components to check the presence of conventional and edge systems by sharing one request result through the react context provider.

# How to test the PR

To test (preparing accounts to test the PR):
    1. Open first the account without any system
    2. Observe that Zero state is shown
    3. Open another account with edge system
    4. Observe that Zero state is not shown
    5. Open another account with conventional system
    6. Observe that Zero state is not shown

# Before the change

# After the change

# Dependent work link

# Checklist:

- [ ] The commit message has the Jira ticket linked
- [ ] PR has a short description
- [ ] Screenshots before and after the change are added
- [ ] Tests for the changes have been added
- [ ] README.md is updated if necessary
- [ ] Needs additional dependent work

### CODE REVIEW

File: src/SmartComponents/HybridInventoryTabs/HybridInventoryTabs.js [hunk 1]
```diff
@@ -13,6 +14,8 @@ const ConventionalSystems = lazy(() =>
 
 const HybridInventory = (props) => {
   const isEdgeParityEnabled = useFeatureFlag('advisor.edge_parity');
+  const { hasEdgeDevices, hasConventionalSystems } =
+    useContext(AccountStatContext);
```
Comment (AsToNlele) [inline src/SmartComponents/HybridInventoryTabs/HybridInventoryTabs.js:18]: Why the need for context? You're not changing the values and only using the defaults

**Review by AsToNlele — APPROVED** [review]
Works as described, goodjob @mkholjuraev ! :smile: 

Had one question

File: src/SmartComponents/HybridInventoryTabs/HybridInventoryTabs.js (hunk 1 above)
Comment (mkholjuraev) [reply src/SmartComponents/HybridInventoryTabs/HybridInventoryTabs.js:18]: Thank you @AsToNlele for the review! Yeah, my bad I forgot to add the context provider that sets context values to be shared across components. Now it is added :)

Comment (mkholjuraev) [reply src/SmartComponents/HybridInventoryTabs/HybridInventoryTabs.js:18]: And my commit ^ dismissed your approval. Please have a quick look one more time

**Pull request merged**